app/models/resturant.rb

```diff
@@ -0,0 +1,8 @@
+class Resturant < ApplicationRecord
```

Comment:
- [ ] **(OPTIONAL)** I don't know if it's intentional, but this might have been a mistake, it's written "Resturant". It's optional because it's your choice really, since it doesn't matter if your app is working.